packages/autorest.python/autorest/codegen/models/operation_group.py

```diff
@@ -80,13 +80,12 @@ def imports(self, async_mode: bool) -> FileImport:
                 operation.imports(async_mode, relative_path=relative_path)
             )
         # for multiapi
-        if not self.code_model.options["version_tolerant"]:
```

Comment:
why is judge logic about `version_tolerant` is removed?

Reply:
The logic inside these criteria takes effect when `models_mode` are not None. I think it's not related to whether of `version_tolerant`.